src/parser/core/Events.js

```diff
@@ -0,0 +1,179 @@
+   * - ability: object of the ability/spell involved. Shape: { name, guid, type: I believe this is the magic school type, abilityIcon }
+   * - resourceActor:
+   * - classResources: array of resources (mana, energy, etc)
+   * - hitPoints:
+   * - maxHitPoints:
+   * - attackPower:
+   * - spellPower:
+   * - x: x location on the map. See paladin/holy/modules/features/MasteryEffectiveness for an example module that uses this data.
+   * - y: y location on the map
+   * - facing: the direction the player is facing
+   * - sourceMarker:
+   * - targetMarker:
+   * - mapID:
+   * - itemLevel:
+   */
+
+  /**
+   * This event is called for events where the player, a player pet or a target of the player/player pet dealt or took damage.
+   * Event specific props:
+   * - amount: effective damage
+   * - absorbed: damage absorbed by a buff on the target (e.g. https://www.wowhead.com/spell=269279/resounding-protection). This should generally be considered effective damage.
+   * - overkill: if the target died, this is the amount of damage that exceeded their remaining health
+   * - hitType:
+   * - mitigated:
+   * - unmitigatedAmount:
+   * - tick:
+   *
+   * NOTE: Do not use this event to track absorb-healing (e.g. by a spell such as Resounding Protection). Use the `absorbed` event instead.
+   * @returns {EventFilter}
+   */
+  get damage() {
+    return new EventFilter('damage');
+  },
+  /**
+   * This event is called for events where the player, a player pet or a target of the player/player pet was healed.
+   * Event specific props:
+   * - amount: effective healing
+   * - absorbed: healing absorbed by a debuff on the target (e.g. https://www.wowhead.com/spell=233263/embrace-of-the-eclipse). This should generally be considered effective healing.
+   * - overheal: overhealing
+   * @returns {EventFilter}
+   */
+  get heal() {
+    return new EventFilter('heal');
+  },
+  /**
+   * Triggered in addition to the regular heal event whenever a heal is absorbed. Can be used to determine what buff or debuff was absorbing the healing.
+   * NOTE: This should only be used if you need to know **which ability soaked the healing**. If you want to track the amount of absorbed healing by a spell, use the `absorb` prop of the `heal` event.
+   * @returns {EventFilter}
+   */
+  get healabsorbed() {
+    return new EventFilter('healabsorbed');
+  },
+  /**
+   * This event is called for events where the player, a player pet or a target of the player/player pet was healed.
+   * Event specific props:
+   * - ability: The ability responsible for absorbed damage (i.e. the shield)
+   * - amount: effective damage absorbed
+   * - attackerID:
+   * - attackerIsFriendly:
+   * - extraAbility: The damage ability that was absorbed
+   * @returns {EventFilter}
+   */
+  get absorbed() {
```

Comment:
This has the same eventfilter as heal(), but the jsdocs are different? I don't deal with heal events so I am not sure if this is intended or not

Reply:
copy paste error